src/containers/LoginScreen/index.js

```diff
@@ -15,12 +16,14 @@ const LoginScreen = ({ navigation }) => {
     user => dispatch(login(user)),
     [dispatch]
   );
+  const isLoading = useLoading('LOGIN');
+
   return (
     <View style={styles.container}>
       <Text style={styles.welcome}>
         {translate('SIGN_IN.title')}
       </Text>
-      <LoginForm onSubmit={loginRequest} />
+      <LoginForm onSubmit={loginRequest} submitting={isLoading} />
```

Comment:
You don't need to add the property `submitting` because `redux-form` is already adding it